libs/src/api/entityManager.ts

```diff
@@ -1,18 +1,32 @@
 import { Base, Services } from './base'
 import type { PlaylistMetadata } from '../services/creatorNode'
-import { Nullable, Utils } from '../utils'
-import {
-  EntityType,
-  Action
-} from '../services/dataContracts/EntityManagerClient'
+import type { Nullable } from '../utils'
 
-export interface PlaylistOperationResponse {
+export enum Action {
+  CREATE = 'Create',
+  UPDATE = 'Update',
+  DELETE = 'Delete',
+  FOLLOW = 'Follow',
+  UNFOLLOW = 'Unfollow',
+  SAVE = 'Save',
+  UNSAVE = 'Unsave',
+  REPOST = 'Repost',
+  UNREPOST = 'Unrepost'
+}
+
+export enum EntityType {
+  TRACK = 'Track',
+  PLAYLIST = 'Playlist',
+  USER = 'User'
+}
+
+export interface EntityManagerOperationResponse {
```

Comment:
for this interface, I think the following in a little more accurate - lmk what you think
```
export interface EntityManagerOperationResponse {
  blockHash: string,
  blockNumber: number,
  error: null
} | {
  blockHash: null,
  blockNumber: null,
  error: string
}
```

Reply:
definitely. i modified to do something similar using type not interface since i couldn't get ^ to work.